### PR TITLE
Fix import into current library issuing wrong message

### LINKED ISSUE #6266: Import into current libarary issues wrong message
This is with version 5.1 (current).

When one chooses "import into current libary" from File-->Import menu and opens a
new bib file and go ahead with import you first get a message "import cancelled" and
a few seconds later "import complete". Import is done properly but the erreneous
message "import cancelled" should not be there.

### PR DESCRIPTION
**Pull request by jcampbell11245**
This change fixes #6266 in which an erroneous message "import canceled" is displayed when importing into the current library. This fix was accomplished by modifying the parameter of the extends clause of the ImportEntriesDialog class to have a return type of Boolean rather than Void and making setResultConverter return false rather than null.

- [x] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by tobiasdiez** [comment]
Welcome to JabRef and thanks for your contribution! 
Can you please describe how your changes fix #6266? I couldn't find the place where the return value of the dialog was used.

**Comment by jcampbell11245** [comment]
Hello, since this issue is a duplicate of #4841 I followed the comments on that issue and implemented [your possible solution](https://github.com/JabRef/jabref/issues/4841#issuecomment-486271379) which resolved the issue. To be honest I do not know exactly how this fixed the issue.

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java [hunk 1]
```diff
@@ -76,14 +76,14 @@ public ImportEntriesDialog(BibDatabaseContext database, BackgroundTask<ParserRes
         Button btn = (Button) this.getDialogPane().lookupButton(importButton);
         btn.disableProperty().bind(booleanBind);
 
-        setResultConverter(button -> {
+       setResultConverter(button -> {
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java:79]: Hi @jcampbell11245, thanks for your contribution.
Tiny issue: Indentation is moved to the left. ;-)

**Comment by jcampbell11245** [comment]
I have pushed a commit fixing the indentation and fixing the problem in the FindUnlinkedFilesDialog.

**Comment by Siedlerchr** [comment]
Thanks again for your contribution! 👍 We hope you are interested in contributing further ;)

**Comment by jcampbell11245** [comment]
You're welcome! This was fun. I hope to contribute further as well.

**Pull request merged**